### ISSUE TITLE
Web Application is not loading after configuring with Vnet Gateway

### ISSUE THREAD

**Issue by Abhishekpathania01** [microsoft/MCW-Migrating-SQL-databases-to-Azure#5]
Exercise 4 [Integrate App Service with the virtual network](https://github.com/microsoft/MCW-Migrating-SQL-databases-to-Azure/blob/master/Hands-on%20lab/HOL%20step-by-step%20-%20Migrating%20SQL%20databases%20to%20Azure.md#exercise-4-integrate-app-service-with-the-virtual-network) After configurung the Vnet with Web Application, web app should load but in my case its not loading. 

Difference is I am having SQL-Mi instance and Vnet Gateway in different RG and Web app is in different RG, we configured everything with same Vnet only ( Regions is same only RG are different )
![image](https://user-images.githubusercontent.com/38715522/64036576-29878100-cb71-11e9-8f53-497182da9fb2.png)

Can you please check this ASAP, I can share lab environment details to quickly check this. 

Thanks
Abhishek

**Comment by Abhishekpathania01** [comment]
Hi @kylebunting as discussed in email the above mentioned issue has been resolved. 
Going forward I am validating the lab further and now I found that there are some steps/screenshot which needs to be updated in the lab-guide due to updates in Azure portal. 

Please review [Exercise 5 Task1](https://github.com/microsoft/MCW-Migrating-SQL-databases-to-Azure/blob/master/Hands-on%20lab/HOL%20step-by-step%20-%20Migrating%20SQL%20databases%20to%20Azure.md#task-1-enable-advanced-data-security) **Enable ADS for all databases on the Managed Instance** we need to click on Vulnerability Assessment option as show in below screenshot:
![image](https://user-images.githubusercontent.com/38715522/64078615-6c289500-ccfa-11e9-8db7-a9dbbd1c219b.png)
 And then configure a storage account, only after that we can enable ADS. Once it is done we'll need to intiate the scan 
![image](https://user-images.githubusercontent.com/38715522/64078631-9ed28d80-ccfa-11e9-9f29-32bb73467b30.png)

Then only we can see the three tiles on the Advanced Data Security blade. 

Please review and update these instructions in the lab guide

Thanks
Abhishek Pathania

**Comment by kylebunting** [comment]
@Abhishekpathania01, I just completed a full test-fix cycle for this lab, and I did not have the same issue with the vulnerability assessment. The storage account is specified during the lab setup and database migration. Perhaps this was an issue because you are using a pre-provisioned SQL MI database, and an associated storage account was not specified? 

We need to be careful about not adding steps to this lab that might not apply if following the lab steps completely, without variations.

I'm going to close this issue. If you run into any other problems, please open a new issue.
